addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -0,0 +1,83 @@
+/** @odoo-module **/
+
+import { TEST_USER_IDS } from '@mail/utils/test_utils';
+import { addModelNamesToFetch, insertModelFields, insertRecords } from '@mail/../tests/helpers/model_definitions_helpers';
+
+//--------------------------------------------------------------------------
+// Model names to fetch
+//--------------------------------------------------------------------------
+
+addModelNamesToFetch([
+    'ir.attachment', 'ir.model', 'ir.model.fields', 'mail.activity', 'mail.activity.type',
+    'mail.channel', 'mail.followers', 'mail.message', 'mail.message.subtype',
+    'mail.notification', 'mail.shortcode', 'mail.template', 'mail.tracking.value',
+    'res.company', 'res.country', 'res.partner', 'res.users', 'res.users.settings',
+]);
+
+//--------------------------------------------------------------------------
+// Insertion of fields
+//--------------------------------------------------------------------------
+
+insertModelFields('mail.activity', {
+    chaining_type: { default: 'suggest' },
+});
+insertModelFields('mail.channel', {
+    author_id: { default: function () { return this.currentPartnerId; } },
```

Comment:
my linter wants function body to a new line (it was already an issue in the mock models file)

Reply:
also could be `default() {` with short syntax